presto-jdbc/src/main/java/io/prestosql/jdbc/AbstractPrestoResultSet.java

```diff
@@ -562,9 +562,11 @@ public Object getObject(int columnIndex)
                 // TODO (https://github.com/prestosql/presto/issues/6048) move to convertFromClientRepresentation
                 return getDate(columnIndex);
             case Types.TIME:
+            case Types.TIME_WITH_TIMEZONE:
```

Comment:
`Fix TIME/TIMESTAMP TZ column declaration for result set` is backward incompatible. Do we care?

Reply:
Yes, it is. I think it's just a bug. I reported this as https://github.com/prestosql/presto/issues/6251 for reference.